.travis.yml

```diff
@@ -1,18 +1,16 @@
 language: ruby
 rvm:
   - jruby
-  - 2.0.0
-  - 2.1
-  - 2.2
-  - 2.3.0
+  - 2.1.10
+  - 2.2.6
+  - 2.3.3
```

Comment:
Ah, sorry forgot about that. I'll play around with travis on my fork and see if I can get the fuzzy version locking playing nicely. For now, I'll go ahead and merge this in.